x-pack/filebeat/module/rabbitmq/_meta/config.yml

```
@@ -0,0 +1,8 @@
+# - module: rabbitmq
+  # All logs
+  # log:
```

```suggestion
  #log:
```
We leave out the space before config option to make commenting out easier and differentiate it from comments. Same on line 4.